float64 x
float64 y
float64 linear_velocity
float64 angular_velocity
string status
